firestore/lite/collection/index.ts

```diff
@@ -47,3 +47,11 @@ export function collectionData<T=DocumentData>(
       }),
   );
 }
+
+export function collectionCountSnap$(query: Query<unknown>): Observable<CountSnapshot> {
```

Comment:
Hmm, thinking about the `$` notation a little more - isn't that technically for Observables, while these are functions that return observables?

So, wouldn't the "Rx-iest" way to do this be:

```js
const count$ = collectionCountSnap()
```

Reply:
I totally missed that when thinking about this. I agree with you and lets strip the `$`.